lib/internal/repl.js

```diff
@@ -10,6 +10,13 @@ const debug = util.debuglog('repl');
 module.exports = Object.create(REPL);
 module.exports.createInternalRepl = createRepl;
 
+const { deprecate } = require('internal/util');
+const _printHistoryDeprecation = deprecate(
+  () => path,
+  'NODE_REPL_HISTORY_FILE is deprecated. Use the plaintext format instead.',
```

Comment:
Might it be more clear to say "use NODE_REPL_HISTORY instead"? That's what the docs say.